src/test/java/com/jcabi/matchers/NoBrokenLinksTest.java

```diff
@@ -77,7 +77,7 @@ public void findsBrLinksInHtmlWithNamespace() throws Exception {
                 )
             ).fetch(),
             Matchers.not(
-                new NoBrokenLinks(new URI("http://www.jcabi.com"))
+                new NoBrokenLinks(new URI("http://google.com"))
```

Comment:
@RomanKisilenko Could you explain the purpose of changing this URI? I'm not sure I understand the change, especially since the title of your pull request refers to additional logging.